Master

mauromascarenhas opened the pull request:
Improvements implemented.

Commit:
New resources added

Added:

- Application icon.
- About section.
- Some controls on menu.
- An agradable visual stylesheet.
- Change view with mouse (Still bugged, need to be reviewed) - Drag
controls.

Improved:

- Visual interface.

Commit:
Merge remote-tracking branch 'origin/master'

Commit:
Improvements and compatibility fixes

Added:
- Help content (functions of buttons).
- Loading screen.

Improved:
- Forms stylesheet.
- Compatibility surface added.
- Compatibility with Unix systems added.

Fixed:
- Cube rotation (substituted trackball by a drag and drop movement
alternative).

Commit:
Lançamento da versão 0.9.0.1 | Release 0.9.0.1

Adicionado:

- Sistema de tradução e arquivos de tradução (português e inglês).
- Sistema de configurações (para salvar preferências de idiomas).
- Imagens da folha de estilo que estavam faltando.

Melhorado:

- Referências e funções não utilizadas removidas.
- Corrigidos erros de linguagem (Frases).

Removido:

- Configurações pessoais de compilação.

>> EN <<

Added:

- Translation system and the translations as well (portuguese and
english).
- Settings system (to save the language preferences).
- Missing stylesheet images.

Improved:

- Removed unused references and functions.
- Fixed language mistakes (Strings).

Removed:

- Personal build configurations.

Commit:
Configurações de compilação | Release settings

Adicionado:

- Configurações de compilação.

>> EN <<

Added:

- Build settings

rafaelpfreire commented on README.md:
```diff
@@ -1,36 +1,28 @@
-# rubix
-Rubix Cube Game developed with QT 5 and OpenGL library
+# RUBIX
```
Mauro, pode retirar esta linha? Minha ideia era tentar deixar multiplataforma.

mauromascarenhas replied:
Esse for Windows é redundante, realmente ele é multiplataforma (Se você olhar o qmake notará que a inclusão da biblioteca será ignorada se não estiver sendo compilado no Win32). Essa linha pode ser retirada tranquilamente.

rafaelpfreire replied:
Humm, agora eu vi. Legal! Acho que podemos então descrever que ele é multiplataforma, ao invés de somente citar o Windows. Também nas seguintes variáveis no qmake, deixe o nome da sua empresa como contribuidora, mas coloque o meu também, por favor.

QMAKE_TARGET_COMPANY = Nintersoft
QMAKE_TARGET_PRODUCT = Rubix
QMAKE_TARGET_DESCRIPTION = Rubix Cube Game 3D! - Remastered
QMAKE_TARGET_COPYRIGHT = Copyright (c) 2016 Nintersoft

mauromascarenhas replied:
Tudo bem, farei as mudanças requeridas.

rafaelpfreire commented on README.md:
```diff
@@ -1,36 +1,28 @@
+O clássico jogo do cubo mágico em 3D! para Windows.
 
-## Controls
+#Aviso importante
```
Acho que esse é um aviso importante, mas pode deixá-lo em inglês?

mauromascarenhas replied:
Sim, se você observar o arquivo Readme original, notará que existem dois idiomas, o arquivo encontra-se totalmente em inglês após a tag EN.

rafaelpfreire replied:
Ah sim, não tinha reparado. Vamos deixar assim então, ficou bom.

mauromascarenhas replied:
Ok

rafaelpfreire commented on README.md:
```diff
@@ -1,36 +1,28 @@
-- d - D : Rotate the Pieces on the Right Face
-- q - Q : Rotate the Pieces on the Back Face
-- f - F : Rotate the Pieces on the Front Face
+#ATENÇÃO: Código Aberto | Licença de terceiros
```
Acho legal deixar esse parágrafo falando da licença, mas sem a parte de ser "clonado". E também em inglês, para ficar igual ao resto

mauromascarenhas replied:
Vou reverter as alterações no README para deixar igual ao seu, daí você dá o merge, e depois eu modifico de novo (isso se você preferir), quanto ao idioma em inglês, ele já está traduzido em dois idiomas.

rafaelpfreire replied:
Ok, deixe nos 2 idiomas então, ficou bom. Acho que essa alteração é muito simples, talvez valha a pena fazer agora já. Assim já matamos isso. Você pode dar um push dessa alteração somente para este repositório, para manter o seu do jeito que está.

mauromascarenhas replied:
Tudo bem, o farei.

rafaelpfreire commented on frmabout.ui:
```diff
@@ -0,0 +1,650 @@
+     </font>
+    </property>
+    <property name="text">
+     <string>Rubix Cube Game 3D! - Remastered</string>
```
Poderia deixar o nome como "Rubix Cube Game 3D!"? Acho que seria legal tirar o "Remastered". Verificar outras partes do código com o nome também.

mauromascarenhas replied:
Ok, se você preferir. Essa modificação será feira assim que possível.

rafaelpfreire commented on mainwindow.ui:
```diff
@@ -1,24 +1,556 @@
-  <property name="windowTitle" >
-   <string>MainWindow</string>
+  <property name="windowTitle">
+   <string>Rubix Cube Game 3D! - Remastered</string>
```
Aqui o Remastered também.

mauromascarenhas replied:
Ok, removerei.

rafaelpfreire reviewed (changes requested):
Mauro, primeiramente muito obrigado pela contribuição! Mas acho que valeria a pena fazermos algumas alterações antes de dar um merge no meu repositório. Deixei alguns comentários no código, o que vc acha?

mauromascarenhas commented:
Ok, vamos verificar os pontos abordados. Estou aguardando as respostas.

mauromascarenhas commented:
Ok, assim que você der uma olhada nas respostas (e respondê-las de novo hehe), poderei efetuar as modificações.

rafaelpfreire commented:
Pronto. Obrigado novamente, Mauro.

mauromascarenhas committed:
Atualizações de dados | Data upgrade

Atualizações solicitadas pelo desenvolvedor do projeto fonte.

EN

Updates requested by the developer of the source program.

mauromascarenhas commented:
Rafael, fiz algumas alterações.
Dá uma olhada pra ver o que você acha.
Qualquer coisa é só chamar.

mauromascarenhas committed:
Visualização | Visualization

Correção na apresentação dos dados

EN

Fixed the data presentation.

mauromascarenhas committed:
README atualizado | updated README

Erros ortográficos corrigidos.

EN

Fixed some orthographic errors.

mauromascarenhas committed:
README atualizado | updated README

Correção de erros ortográficos.

EN

Orthographic error fix.
